distribution/solr-distro/src/main/resources/security/solr-default.policy

```diff
@@ -0,0 +1,51 @@
+priority "grant";
+
+deny {
+    // Read, write, or execute any file
+    permission java.io.FilePermission "<<ALL FILES>>", "read, write, execute, delete";
+
+    permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
+    permission java.util.PropertyPermission "java.io.tmpdir", "write";
+    permission java.util.PropertyPermission "user.home", "write";
+
+    // Change current security manager
+    permission java.lang.RuntimePermission "setSecurityManager";
+
+    // Modify application permissions at will.
+    permission java.security.SecurityPermission "getDomainCombiner";
+    permission java.security.SecurityPermission "setPolicy";
+    permission java.security.SecurityPermission "insertProvider";
+    permission java.security.SecurityPermission "removeProvider.*";
+    permission java.security.SecurityPermission "clearProviderProperties.*";
+    permission java.security.SecurityPermission "removeProviderProperty.*";
+
+    // Load classes into any protection domain
+    permission java.lang.RuntimePermission "createClassLoader";
+}
+
+grant {
+    // Solr folder access permissions
+    permission java.io.FilePermission "..", "read, write, delete";
```

Comment:
❓ Are these going to be relative to this directory or to the directory from which the user launches the application?

Reply:
I tested to make sure that it it stays relative to the location of the solr start.jar itself which is at `ddf_home/solr/server/start.jar`, so it gives permission to anything under `ddf_home/solr`